components/FAQ/molecules/Collage.styles.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+
+import { Green100, Grey010 } from 'components/styling/colors';
+
+export const CollageContainer = styled.div`
+  margin: 30px 0 30px -40px;
+  grid-area: collage;
+`;
+
+export const LargerSquare = styled.div`
+  height: 246px;
+  width: 246px;
+  background-color: ${Grey010};
+
+  @media screen and (min-width: 1200px) {
+    height: 510px;
+    width: 510px;
+  }
+
+  @media screen and (min-width: 1400px) {
+    position: relative;
+    left: -90px;
+  }
+`;
+
+export const SmallerSquare = styled.div`
+  height: 148px;
+  width: 148px;
+  background-color: ${Green100};
+  position: relative;
+  top: 124px;
+  
+  @media screen and (min-width: 1200px) {
+    height: 305px;
+    width: 305px;
+    top: 255px;
+  }
+`;
+
+export const ImageContainer = styled.div`
+  width: 272px;
+  height: 160px;
+  position: relative;
+  left: 64px;
+  bottom: 113px;
+
+  @media screen and (min-width: 1200px) {
```

Comment:
I'd rather go with % values here with a min-width, or min calculation here. Looks weird on tablets IMO.

https://developer.mozilla.org/en-US/docs/Web/CSS/min

Reply:
do you mean it is too small or related to the page width?

Reply:
changed this a bit, please review latest push @mikitovicsmark